extension/src/popup/components/BottomNav/index.tsx

```diff
@@ -1,17 +1,40 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
+
+import { ROUTES } from "popup/constants/routes";
 
 import HistoryIcon from "popup/assets/icon-history.svg";
-import SwapIcon from "popup/assets/icon-swap.svg";
```

Comment:
I noticed that Charles accidentally left the Swap icon in the bottom nav in a few screens. Swap will actually be coming in a later version, so I omitted for now!